pkg/config/filter.go

```diff
@@ -29,6 +30,26 @@ func (f Filters) Merge(f2 Filters) {
 	}
 }
 
+type FilterFailed struct{
+	Filter string //?
+	Fatal bool
+	Err error
+}
```

Comment:
Filter is the filter that caused the Failure,  
Fatal is if the error should cause a fatal error,  
Err is a encapsulated error that is of value to the caller